daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Classify.scala

```diff
@@ -105,14 +110,13 @@ object Classify { // classify the machine state w.r.t what step occurs next
 
   def classifyKont(kont: Kont, counts: Counts): Unit = {
     kont match {
-      case KPop(_) => counts.kpop += 1
-      case KArg(_) => counts.karg += 1
+      case KArg(_, _, _) => counts.karg += 1
       case KFun(_, _, _) => counts.kfun += 1
-      case KPushTo(_, _) => counts.kpushto += 1
+      case KPushTo(_, _, _, _) => counts.kpushto += 1
```

Comment:
Yes. Totally agree. I also realised this would be a better coding idiom here.
Nearly made the change.
Had to stop myself as it it clearly an unrelated refactoring.